testing/libspatialite/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=libspatialite
+pkgver=4.3.0
+_pkgver="$pkgver"a
+pkgrel=0
+pkgdesc="SpatiaLite is an open source library intended to extend the SQLite core to support fully fledged Spatial SQL capabilities."
+url="https://www.gaia-gis.it/fossil/libspatialite/index"
+arch="all"
+license="MPLv1.1"
+options="!check"
+makedepends="automake geos-dev sqlite-dev proj4-dev zlib-dev libxml2-dev"
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::http://www.gaia-gis.it/gaia-sins/$pkgname-$_pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$_pkgver"
+
+build() {
+        cd "$builddir"
+
+        ./configure \
+                -prefix=/usr \
```

Comment:
This should be `--prefix` ?
We also normally have `--build=$CBUILD` and `--host=$CHOST`.

Reply:
I'm sure I inadvertently removed the missing `-`. Surprised I didn't notice this earlier, though perhaps it just resulted in a warning that I overlooked.

As for `--build` and `--host`, setting these result in errors:
```
checking build system type... Invalid configuration `x86_64-alpine-linux-musl': machine `x86_64-alpine-linux' not recognized
checking host system type... Invalid configuration `x86_64-alpine-linux-musl': machine `x86_64-alpine-linux' not recognized
```

Things build fine without.

Reply:
They are needed for potential cross build support. The above error is fixed by using `update_config_sub` in `prepare`.